support-workers/src/it/resources/logback-test.xml

```diff
@@ -1,3 +1,20 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Turn off logging during tests -->
```

Comment:
not sure why the logging was off, as it was necessary to work out what was going on.  I suppose I can turn on locally and turn off before pushing, but given we only ever run locally the logging would be nice.